Aptoide/appcoins-wallet-android
# [APPC-1191] Added more use cases on get campaign mapping

**What does this PR do?**

 This ticket adds one more use case to the mapping of the get campaign response, as well as fixing the response bundle as it was not mapping the correct response

**Where should the reviewer start?**

-CampaignService.kt

**How should this be manually tested?**

With a wallet that already converted for that packageName
-A notification saying you were already rewarded should appear (with or without campaign)
With a wallet that has not yet converted for that packageName
-If there’s a campaign: It should start the PoA and in the end of the 12 proofs not give an no success notification
-If there’s not a campaign: A notification saying that there’s no campaign available should appear (before it was saying that you were already rewarded)

Also think of other use cases that we may have forgotten 

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1191

**Code Review Checklist**
